test/src/fake_datastore.dart

```diff
@@ -0,0 +1,306 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// This is copied from https://github.com/flutter/cocoon/blob/master/app_dart/test/src/datastore/fake_datastore.dart
```

Comment:
This is good here. Although it seems to be better if we published that `fake_datastore` as a pub package so we can just use it by adding a dependency here. That way, we'll get the updated code if someone improved or fixed the `fake_datastore` code.

Reply:
It's better to consider that when we productionize metrics_center. It's out of scope of this PR right now.